server/serverConstants.js

```diff
@@ -23,6 +23,8 @@ E.REDIS_PHONE_PREFIX = 'otrtc_phone_';
 
 E.REDIS_ROOM_MATCHES = `${E.REDIS_ROOM_PREFIX}*`;
 
+E.SPEECH_TO_TEXT = { envVar: 'SPEECH_TO_TEXT', jsonPath: 'SpeechToText.url', required: true };
```

Comment:
please rename this to SPEECH_TO_TEXT_SERVER_URL or something similar. i think that SPEECH_TO_TEXT is not descriptive enough